x-pack/test/security_solution_cypress/cypress/tasks/select_all_spaces.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { SPACE_SELECTOR_COMBO_BOX } from '../screens/custom_roles/assign_to_space_flyout';
+
+export const selectAllSpaces = (): void => {
+  cy.get('[data-test-subj="addSpacePrivilegeButton"]').click();
+
+  // select all spaces
+  cy.get(SPACE_SELECTOR_COMBO_BOX).click();
+  cy.get(SPACE_SELECTOR_COMBO_BOX).type('* All Spaces');
+  cy.get(SPACE_SELECTOR_COMBO_BOX).type('{enter}');
+
+  // expand security privileges
+  cy.get('[data-test-subj="featureCategory_securitySolution"]').click();
```

Comment:
NIT: Same as: https://github.com/elastic/kibana/pull/215334/files#r2010209897

Reply:
Oh yeah, I missed that looks like in the commits.

Reply:
Done 073f166ee8363d4de11c55823ff625caa9578277